### PR TITLE
[pr-deploy] Add README.md

### PR DESCRIPTION
**Pull request by estherkim**
Related to https://github.com/ampproject/amp-github-apps/issues/357

### CODE REVIEW

File: pr-deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+AMP PR Deploy Bot
+===================
+
+A GitHub App deploys a PR branch to a static website when you ask it to.
```
Comment (rcebulko) [inline pr-deploy/README.md]: Nit: "A GitHub app *which* deploys"

File: pr-deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+AMP PR Deploy Bot
+===================
+
+A GitHub App deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1). <br>
```
Comment (rcebulko) [inline pr-deploy/README.md]: `<br>` in Markdown?

File: pr-deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+The app is hosted by [Google App Engine](https://amp-pr-deploy-bot.appspot.com) and is installed exclusively on the ampproject/amphtml repository.
+
+How the app works
+----------------
+1. A commit is pushed to a new or existing pull request on ampproject/amphtml.
+2. Travis compiles your changes and uploads the build artifacts and example pages as `amp_dist_<Travis build number>.zip` to a remote storage location. During this step, a check called `ampproject/pr-deploy` is set to `pending`.
+3. a) If there was a compilation error, Travis tells the AMP PR Deploy Bot that there's nothing left to do until the error is fixed. <br>
+   b) If there were no errors, Travis tells the AMP PR Deploy Bot that a test site is ready to be deployed. `ampproject/pr-deploy` is now `netural`
+4. A test site is deployed by clicking the 'Deploy Me' button in the details page of `ampproject/pr-deploy`. The app unzips and writes `amp_dist_<Travis build number>.zip` to the public Google Cloud Storage bucket.
+5. `ampproject/pr-deploy` completes with the website URL. `https://console.cloud.`
```
Comment (rcebulko) [inline pr-deploy/README.md]: What is `https://console.cloud.`?

File: pr-deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Here's a quick [demo](https://github.com/ampproject/amphtml/pull/24274) on how to create a test site.
+
+Local development
+-----------------
+
+1. Clone this repository and cd into the `pr-deploy` directory.
+2. `npm install`
+3. Start a new [Smee channel](https://smee.io/). This can be used to proxy
```
Comment (rcebulko) [inline pr-deploy/README.md]: General comment: Similar instructions exist in other set-up guides for Probot apps; we should potentially mark this down somewhere for unifying docs around these app.

Also, this skips `4` :smile:

File: pr-deploy/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+5. Create a new [GitHub App](https://github.com/settings/apps/new) with the following settings:
+   * General
+     * Set _Homepage URL_ to the Smee channel
+     * Set _Webhook URL_ to the Smee channel
+     * Set _Webhook Secret_ to any pin your choice
+   * Permissions and Events
+     * Set _Checks_ to Read & write
+     * Set _Metadata_ to Read-only
+     * Set _Pull requests_ to Read-only
+     * Subscribe to events: _Check run_, _Pull request_
```
Comment (rcebulko) [inline pr-deploy/README.md]: Nice!

File: pr-deploy/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+     * Set _Webhook Secret_ to any pin your choice
+   * Permissions and Events
+     * Set _Checks_ to Read & write
+     * Set _Metadata_ to Read-only
+     * Set _Pull requests_ to Read-only
+     * Subscribe to events: _Check run_, _Pull request_   
+6. After creating the application, generate and download a private key. Also
+   take note of the App ID.
+7. Create a personal access token belonging to a GitHub _user_ with the
+   `public_repo` and `read:org` permissions and nore its access token.
```
Comment (rcebulko) [inline pr-deploy/README.md]: Typo: `nore`

File: pr-deploy/README.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+===================
+
+A GitHub App deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1). <br>
+The app is hosted by [Google App Engine](https://amp-pr-deploy-bot.appspot.com) and is installed exclusively on the ampproject/amphtml repository.
+
+How the app works
+----------------
+1. A commit is pushed to a new or existing pull request on ampproject/amphtml.
```
Comment (danielrozenberg) [inline pr-deploy/README.md]: Supernit: I don't like periods at the end of lists :P

File: pr-deploy/README.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+     * Set _Checks_ to Read & write
+     * Set _Metadata_ to Read-only
+     * Set _Pull requests_ to Read-only
+     * Subscribe to events: _Check run_, _Pull request_   
+6. After creating the application, generate and download a private key. Also
+   take note of the App ID.
+7. Create a personal access token belonging to a GitHub _user_ with the
+   `public_repo` and `read:org` permissions and nore its access token.
+8. Install the application on a GitHub repository that you want to use for
+   testing. You might want to create a new repository for this purpose.
```
Comment (danielrozenberg) [inline pr-deploy/README.md]: "You might want to fork the [ampproject/amphtml](https://github.com/ampproject/amphtml) repository or create a new one for this purpose"

File: pr-deploy/README.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+9. Copy the `.env.example` file to `.env` and modify the fields based on the
+   instructions in that file and the values from the GitHub App page:
+   * The value for the `PRIVATE_KEY` field is a base64 representation of the
+     `.pem` file you downloaded from the GitHub App page in Step 6. On Linux/Mac you can
```
Comment (danielrozenberg) [inline pr-deploy/README.md]: Step 5

File: pr-deploy/README.md (hunk 7 above)
Comment (rcebulko) [reply pr-deploy/README.md]: Counter-argument: When list items contain multiple sentences, I think they're reasonable

File: pr-deploy/README.md (hunk 2 above)
Comment (estherkim) [reply pr-deploy/README.md]: yep, I didn't want an extra line between the two.

File: pr-deploy/README.md (hunk 7 above)
Comment (estherkim) [reply pr-deploy/README.md]: Yeah I'm gonna vote keeping the periods.

Comment (danielrozenberg) [reply pr-deploy/README.md]: 😢

File: pr-deploy/README.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+A GitHub App deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1). <br>
+The app is hosted by [Google App Engine](https://amp-pr-deploy-bot.appspot.com) and is installed exclusively on the ampproject/amphtml repository.
+
+How the app works
+----------------
+1. A commit is pushed to a new or existing pull request on ampproject/amphtml.
+2. Travis compiles your changes and uploads the build artifacts and example pages as `amp_dist_<Travis build number>.zip` to a remote storage location. During this step, a check called `ampproject/pr-deploy` is set to `pending`.
+3. a) If there was a compilation error, Travis tells the AMP PR Deploy Bot that there's nothing left to do until the error is fixed. <br>
```
Comment (rcebulko) [inline pr-deploy/README.md]: IIRC if you put the a/b on new lines with an indent they will render nested in Markdown, but not sure

**Pull request merged**